PrebidMobile/ConfigurationAndTargeting/Targeting.swift

```diff
@@ -303,6 +303,20 @@ public class Targeting: NSObject {
     
     // MARK: - Public Methods
     
+    // MARK: Arbitrary ORTB Configuration
+    
+    /// Sets the global-level OpenRTB configuration string.
+    ///
+    /// - Parameter ortbObject: The global-level OpenRTB configuration string to set. Can be `nil` to clear the configuration.
+    public func setGlobalORTBConfig(_ ortbConfig: String?) {
```

Comment:
Please add an example of how this method is used in the demo apps. 

Let's set: 
- `ext.myext.test = 1` 
- `displaymanager` = Google
- `displaymanagerver` = $CurrentGMASDKVersion